tests/test_cli/test_exit_transaction_keystore.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+from click.testing import CliRunner
+
+from staking_deposit.credentials import Credential
+from staking_deposit.deposit import cli
+from staking_deposit.settings import get_chain_setting
+from staking_deposit.utils.constants import DEFAULT_EXIT_TRANSACTION_FOLDER_NAME
+
+from tests.test_cli.helpers import (
+    clean_exit_transaction_folder,
+    read_json_file,
+    verify_file_permission,
+)
+
+
+def test_exit_transaction_menmonic() -> None:
```

Comment:
We are only testing the happy and successful path for exit-transaction-keystore. It would be nice to add a few tests for the paths where it fail.

Reply:
Also the name is wrong :D

Going to add a couple of edge cases here.

One case that I don't think is possible to test is chain mismatch. If you create a validator on holesky and use that keystore to generate an exit message with mainnet specified, it will create one, though invalid.  If you know of a way to verify the chain, let me know and I'll add logic for it!

Reply:
From my understanding, the signed exit message depends on the `fork_version` and `genesis_validators_root` for the domain. We could test this by generating a different exit message for all the of testnets and testing that it fails with the `verify_signed_exit_json`/`validate_signed_exit` functions by passing the mainnet chain_settings for instance. That would test the chain mismatch.

Reply:
My bad. I think you cannot know for which chain a keystore was made for so that part cannot be tested, ie. a validator deposited on holekey, using that validator keystore and creating an exit message for mainnet. That exit message should always fail when broadcasted.